### PR TITLE
chore: upgrade dependencies to latest version

### PR DESCRIPTION
**Pull request by ersachin3112**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [x] **metadata update**

### Motivation / Use-Case
upgrade dependencies to latest version

### Breaking Changes
No

### Additional Info
No

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "dist"
   ],
   "peerDependencies": {
-    "@rspack/core": "0.x || 1.x",
+    "@rspack/core": "^0.5.5",
```
Comment (alexander-akait) [inline package.json:46]: Please don't do it

Comment (ersachin3112) [reply package.json:46]: ok got it..
 i think it,s a needed one

File: package.json [hunk 2]
```diff
@@ -43,31 +43,27 @@
     "dist"
   ],
   "peerDependencies": {
-    "@rspack/core": "0.x || 1.x",
```
Comment (alexander-akait) [inline package.json:46]: You should not remove it from here

**Comment by alexander-akait** [comment]
Fixed in master

**Pull request closed without merging**